### PR TITLE
Added FW Versions for 2022 Sonata.

### PR DESCRIPTION
**Pull request by humza2000**
Added for CAR.SONATA :

Ecu.engine : 
b'\xf1\x870\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf1\x81HM6M1_0a0_J10'

Ecu.transmission: 
b'\xf1\x00T02601BL  T02832A1  VDN8T25XXX832NS8G\x0e\xfeE'

b'\xf1\x87954A02N060\x00\x00\x00\x00\x00\xf1\x81T02832A1  \xf1\x00T02601BL  T02832A1  VDN8T25XXX832NS8G\x0e\xfeE'

b'\xf1\x87954A02N060\x00\x00\x00\x00\x00\xf1\x82VDN8T25XXX832NS8'

### CODE REVIEW

**Comment by humza2000** [comment]
Second commit was to fix a typo.

**Comment by sshane** [comment]
Can you provide a route where these came from

**Comment by humza2000** [comment]
Sonata firmwares came from ba75a2db7d699895|2022-11-08--20-35-04.

I used to drive an Elantra with a banned comma 2 both of which I don't own anymore. I had added these FW versions to the comma 2 and they did work.

**Comment by sshane** [comment]
Let's only add FW for what we can verify with a route.

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -512,7 +513,10 @@ class Buttons:
       b'\xf1\x00HT6WA250BLHT6WA910A1SDN8G25NB1\x00\x00\x00\x00\x00\x00\x96\xa1\xf1\x92',
       b'\xf1\x00HT6WA280BLHT6WAD10A1SDN8G25NB2\x00\x00\x00\x00\x00\x00\x08\xc9O:',
       b'\xf1\x00T02601BL  T02730A1  VDN8T25XXX730NS5\xf7_\x92\xf5',
+      b'\xf1\x00T02601BL  T02832A1  VDN8T25XXX832NS8G\x0e\xfeE',
       b'\xf1\x87954A02N060\x00\x00\x00\x00\x00\xf1\x81T02730A1  \xf1\x00T02601BL  T02730A1  VDN8T25XXX730NS5\xf7_\x92\xf5',
+      b'\xf1\x87954A02N060\x00\x00\x00\x00\x00\xf1\x81T02832A1  \xf1\x00T02601BL  T02832A1  VDN8T25XXX832NS8G\x0e\xfeE',
+      b'\xf1\x87954A02N060\x00\x00\x00\x00\x00\xf1\x82VDN8T25XXX832NS8',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:519]: Make sure you only add FW from Hyundai. This is a VW response

Comment (sshane) [inline selfdrive/car/hyundai/values.py:519]:
```suggestion
```

**Comment by sshane** [comment]
Moving to https://github.com/commaai/openpilot/pull/26428, can't edit PR

**Pull request closed without merging**